hcx-core/hcx-common/src/main/java/org/swasth/common/exception/ClientException.java

```diff
@@ -0,0 +1,8 @@
+package org.swasth.common.exception;
+
+public class ClientException extends Exception{
+
+    public ClientException(String message) {
```

Comment:
Define errorCode also for the exception.

Reply:
Done